koji-containerbuild.spec

```diff
@@ -36,12 +36,13 @@ Source0:        %{name}-%{version}.tar.gz
 BuildArch:      noarch
 BuildRoot:      %{_tmppath}/%{name}-%{version}-%{release}-root-%(%{__id_u} -n)
 
-BuildRequires:  python
-BuildRequires:  python2-devel
 %if 0%{with python3}
 BuildRequires:  python3-devel
-%endif
+BuildRequires:  python3-setuptools
+%else
```

Comment:
This means we stop building both py2 and py3 packages for Fedora, I don't think that we agreed on dropping py2 packages from fedora

Reply:
Was agreed that we drop py2 from fedora